doc/Language/5to6-perlop.pod6

```diff
@@ -161,10 +161,10 @@ In list context, C<..> operates as the range operator and should not
 need to be changed. That said, there are exclusionary range operators
 that may be useful. These are:
 
-    infix  ..^  Excludes endpoint. C<1..^5; # 1 2 3 4 >
-    infix  ^..  Excludes startpoint. C<1^..5; # 2 3 4 5>
-    infix  ^..^ Excludes both start- and endpoint. C<1^..^5; # 2 3 4>
-    prefix ^    From 0, excludes endpoint. C<^5; # 0 1 2 3 4>
+    infix  ..^  Excludes endpoint. C<1..^5>; # 1 2 3 4 >
```

Comment:
Correct, shame on me I didn't look at the ending bracket.
I suspect the solution is to remove the C<> inline block at all. 
Is it possible to cherry pick this one? <https://github.com/perl6/doc/commit/a19a6b775f4c3391bb14280d891e8a7ecde4eed3>